guidelines.md

```diff
@@ -76,7 +76,80 @@ The enterprise is based in UK, it accepts `GBP`, `EUR` and `USD` currencies \(an
 
 * **PlatformAddress** - `https://www.mews.li`
 * **ClientToken** - Will be provided to you by our integration team. For further information, please contact [integrations@mewssystems.com](mailto://integrations@mewssystems.com).
-* **AccessToken** - Depends on the enterprise, should be provided to you by the enterprise admin.
+* **AccessToken** - Will be provided to you via [email and your provided webhook](guidelines.md#AccessToken-delivery).
```

Comment:
As commented in other PR, certification process should not be in guidelines. delivery via email should be part of certification section. delivery via webhook as part of webhook section. so two links here.

Reply:
This can be done in the other PR when we introduce certification information to documentation. for now there has been no changes to that